src/ResourceManager/Compute/Commands.Compute/VirtualMachine/VirtualMachineBaseCmdlet.cs

```diff
@@ -186,6 +186,7 @@ private static int GetTabLength(Object obj, int max, int depth, List<Tuple<strin
                                 else
                                 {
                                     tupleList.Add(MakeTuple(property.Name, GetChildProperties(childObject), depth));
+                                    max = Math.Max(max, property.Name.Length);
```

Comment:
@hyonholee so this change is in the `GetTabLength` method, which is called from `FormatObject`, but where is `FormatObject` used?